src/com/google/enterprise/adaptor/sharepoint/SharePointAdaptor.java

```diff
@@ -1043,7 +1046,8 @@ private void getDocIdsSiteCollectionOnly(DocIdPusher pusher)
     Map<GroupPrincipal, Collection<Principal>> groupDefs 
         = new HashMap<GroupPrincipal, Collection<Principal>>();
     groupDefs.putAll(scAdaptor.computeMembersForGroups(site.getGroups()));
-    pusher.pushGroupDefinitions(groupDefs, false);
+    String siteId = site.getMetadata().getID();
```

Comment:
It looks like you're always doing this before calling sitePushGroupDefinitions. Maybe just pass the site itself?

Reply:
As discussed, passing only string (siteId) instead of site object.